src/app/modules/timesheet/services/timesheet.service.ts

```diff
@@ -120,14 +120,44 @@ export class TimesheetService {
   }
 
   public openLastTimesheetInLocal(): boolean {
-    const timesheetsOfLocalStorage = this.getTimesheetsLocal();
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
     if (timesheetsOfLocalStorage.length > 0) {
       this.setTimesheet(this._localSaveService.getLocalItem(timesheetsOfLocalStorage[timesheetsOfLocalStorage.length - 1]));
       return true;
     }
     return false;
   }
 
+  public getPresentTimesheet(transferedTimesheet: Timesheet) {
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
+    const localStorageSize = this.getLocalstorageTimesheetsList().length;
```

Comment:
`localStorageTimesheetsListSize` ? Yes it is long but now the length of the variable names almost no longer influences the performance :)